thirdeye/thirdeye-frontend/app/pods/manage/alert/edit/controller.js

```diff
@@ -516,21 +453,48 @@ export default Controller.extend({
      * @returns {Promise}
      */
     onSubmit() {
-      const currentId = this.get('alertId');
-      const postFunctionBody = this.get('model.alertData');
-      const newGroupName = this.get('newConfigGroupName');
-      const newEmails = this.get('alertGroupNewRecipient');
-      const oldEmails = this.get('selectedConfigGroupRecipients');
+      const {
+        alertData: postFunctionBody,
+        originalConfigGroup
+      } = this.model;
```

Comment:
do we need to use `this.get('model')` here? do we expect it to update?

Reply:
no - not needed. I'll modify.